src/main/java/io/jenkins/plugins/analysis/core/util/QualityGateStatus.java

```diff
@@ -48,16 +48,23 @@ public boolean isSuccessful() {
         return this == PASSED || this == INACTIVE;
     }
 
+    /**
+     * Returns the associated {@link Result}.
+     *
+     * @return the associated {@link Result}
+     */
+    public Result getResult() {
+        return result;
+    }
+
     /**
      * Sets the result of the specified run to the associated value of this quality gate status.
      *
      * @param run
      *         the run to set the result for
      */
     public void setResult(final Run<?, ?> run) {
```

Comment:
Wouldn't it make sense to remove the method at all?

Reply:
Ok I will remove it, my only concern was that it was a public method in a public class and I wasn't sure if anyone might be calling it directly elsewhere, but I would be surprised if another plugin depended on this one.

Reply:
Yes, I see it in the same way. Some people want to migrate their plugins from analysis-core to warnings-ng, but they will not use those methods.

Reply:
Done in 8209b48